virt/virtual_machines/virt-automating-windows-sysprep.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="virt-automating-windows-sysprep"]
+= Automating Windows installation with sysprep
+include::modules/virt-document-attributes.adoc[]
+:context: virt-automating-windows-sysprep
+
+toc::[]
+
+You can use Microsoft DVD images and `sysprep` to automate Windows installation, setup, and software provisioning for virtual machines.
```

Comment:
"to automate Windows installation, setup, and software provisioning for virtual machines."

I think it might be clearer if you phrased it like this: "to automate the installation, setup, and software provisioning of Windows virtual machines."